src/test/java/com/linagora/dav/contracts/CalJsonContract.java

```diff
@@ -281,197 +287,4368 @@ void reportShouldListEvents() {
             .request(HttpMethod.valueOf("REPORT"))
             .uri("/calendars/" + testUser.id() + "/" + testUser.id() + ".json")
             .send(body("""
-                {"match":{"start":"20250201T000000","end":"20250215T000000"}}""")));
+                {"match":{"start":"20250201T000000","end":"20250215T000000"}}""")));
+
+        assertThatJson(response.body())
+            .whenIgnoringPaths("_embedded.dav:item[0].data[1][0][3]")  // ignore prodid
+            .isEqualTo(String.format("""
+                {
+                    "_links": {
+                        "self": {
+                            "href": "/calendars/%s/%s.json"
+                        }
+                    },
+                    "_embedded": {
+                        "dav:item": [
+                            {
+                                "_links": {
+                                    "self": {
+                                        "href": "/calendars/%s/%s/abcd.ics"
+                                    }
+                                },
+                                "etag": "\\"8c97fb06c60212c47d46a9c8c0f625ef\\"",
+                                "data": [
+                                    "vcalendar",
+                                    [
+                                        [
+                                            "prodid",
+                                            {},
+                                            "text",
+                                            "-//Sabre//Sabre VObject 4.1.3//EN"
+                                        ],
+                                        [
+                                            "version",
+                                            {},
+                                            "text",
+                                            "2.0"
+                                        ],
+                                        [
+                                            "calscale",
+                                            {},
+                                            "text",
+                                            "GREGORIAN"
+                                        ]
+                                    ],
+                                    [
+                                        [
+                                            "vevent",
+                                            [
+                                                [
+                                                    "uid",
+                                                    {},
+                                                    "text",
+                                                    "47d90176-b477-4fe1-91b3-a36ec0cfc67b"
+                                                ],
```

Comment:
```suggestion
                                                [ "uid", {}, "text",  "47d90176-b477-4fe1-91b3-a36ec0cfc67b"],
```

Let's adopt a compact form for those JSON

It would make reading way easier

Reply:
Here and everywhere